### PR TITLE
Migration Installer

### PR DESCRIPTION
**Pull request by solverat**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #89

### CODE REVIEW

File: src/ToolboxBundle/Migrations/Version20190213200859.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class Version20190213200859 extends AbstractPimcoreMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $fileSystem = new Filesystem();
+        if ($fileSystem->exists(Install::SYSTEM_CONFIG_DIR_PATH . '/config.yml')) {
+            $fileSystem->remove(Install::SYSTEM_CONFIG_DIR_PATH . '/config.yml');
```
Comment (dpfaffenbauer) [inline src/ToolboxBundle/Migrations/Version20190213200859.php]: FINALLY!!!!!!! :D

File: src/ToolboxBundle/Migrations/Version00000001.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+class Version00000001 extends AbstractPimcoreMigration
+{
+    /**
+     * @return bool
+     */
+    public function doesSqlMigrations(): bool
```
Comment (scrummer) [inline src/ToolboxBundle/Migrations/Version00000001.php]: Pimcore 5.4 compatibility?

Comment (solverat) [reply src/ToolboxBundle/Migrations/Version00000001.php]: true story. will do that in a separate pr. thanks!

Comment (scrummer) [reply src/ToolboxBundle/Migrations/Version00000001.php]: NVM - all return types are already available in 7.1 :)

http://php.net/manual/de/migration71.new-features.php

Comment (solverat) [reply src/ToolboxBundle/Migrations/Version00000001.php]: no problem here, pimcore 5.4 requires php >=7.1 which supports return types like those.

**Pull request merged**